addons/mass_mailing/models/mailing.py

```diff
@@ -1199,3 +1250,29 @@ def _image_to_url(b64image: bytes):
             return lxml.html.tostring(root)
 
         return body_html
+
+    @api.model
+    def get_favorites(self):
+        """Return all mailing set as favorite and skip mailing with empty body."""
+        values_list = self.search_read(
+            domain=[
+                ('body_arch', '!=', False),
```

Comment:
Technically this does not filter out false-void bodies (like ``<p><br /></p>``). I would suggest to fetch all favorites (we should not have thousands of them), then filter out void mailings using the ``is_body_empty`` field. Other solution would be to do an sql search to filter out pseudo-void content in db, seems harsh.